ding_unilogin/src/Exception/HttpBadRequestException.php

```diff
@@ -3,7 +3,7 @@
 namespace Drupal\ding_unilogin\Exception;
 
 /**
- * Class HttpBadRequestException.
+ * Http Bad Request Exception class.
```

Comment:
Why is the class name spaced? Same ? for the other spaces class names?

Reply:
I was trying to shut up `phpcs` complaining about ”The class short comment should describe what the class does and not simply repeat the class name (Drupal.Commenting.ClassComment.Short)”. Now I've excluded this rule in `phpcs.xml`.